src/AccessibilityInsights.Extensions.AzureDevOps/Models/ExtensionConfiguration.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Newtonsoft.Json;
+
+namespace AccessibilityInsights.Extensions.AzureDevOps.Models
+{
+    public class ExtensionConfiguration
+    {
+        public int ZoomLevel { get; set; }
+
+        public ConnectionInfo SavedConnection { get; set; }
+
+        public ConnectionCache CachedConnections { get; set; }
+
+        public string GetSerializedConfig() => JsonConvert.SerializeObject(this);
+
+        public void LoadFromSerializedString(string serializedConfig)
+        {
+            var config = JsonConvert.DeserializeObject<ExtensionConfiguration>(serializedConfig);
```

Comment:
>config [](start = 16, length = 6)

Probably worth some error handling here in case the serialized data is unparsable

Reply:
Errors now caught--I don't think we want to do anything with the exception